.travis.yml

```diff
@@ -50,41 +50,43 @@ before_install:
   - git fetch --unshallow
   # Install libraries required for protobuf generation
   - |
-    if [ $PHASE = "build_checkstyle_protobuf" ]; then \
-      sudo apt-get install -y libboost-all-dev libzookeeper-mt-dev libsasl2-dev cmake libcppunit-dev checkinstall && \
-      pushd .. && \
-      if [ -f $HOME/protobuf/protobuf_3.16.1* ]; then \
-        sudo dpkg -i $HOME/protobuf/protobuf_3.16.1*; \
-      else \
-        wget https://github.com/protocolbuffers/protobuf/releases/download/v3.16.1/protobuf-java-3.16.1.zip && \
-        unzip protobuf-java-3.16.1.zip && pushd protobuf-3.16.1 && \
-        ./configure && sudo make && sudo checkinstall -y && \
-        if [ ! -d $HOME/protobuf ]; then \
-          mkdir -p $HOME/protobuf; \
-        fi && \
-        mv protobuf_3.16.1* $HOME/protobuf/ && popd; \
-      fi && \
-      sudo ldconfig && popd; \
+    if [ $PHASE = "build_checkstyle_protobuf" ]; then
+      sudo apt-get install -y libboost-all-dev libzookeeper-mt-dev libsasl2-dev cmake libcppunit-dev checkinstall &&
+      pushd .. &&
+      if [ -f $HOME/protobuf/protobuf_3.16.1* ]; then
+        sudo dpkg -i $HOME/protobuf/protobuf_3.16.1*;
+      else
+        wget https://github.com/protocolbuffers/protobuf/releases/download/v3.16.1/protobuf-java-3.16.1.zip &&
+        unzip protobuf-java-3.16.1.zip && pushd protobuf-3.16.1 &&
+        ./configure && sudo make && sudo checkinstall -y &&
+        if [ ! -d $HOME/protobuf ]; then
+          mkdir -p $HOME/protobuf;
+        fi &&
+        mv protobuf_3.16.1* $HOME/protobuf/ && popd;
+      fi &&
+      sudo ldconfig && popd;
     fi
 install:
   # For tests phase runs unit tests
   # For protobuf phase: builds Drill project, performs license checkstyle goal and regenerates Java and C++ Protobuf files
   - |
-    if [ $PHASE = "tests" ]; then \
-      mvn install --batch-mode --no-transfer-progress \
-        -DexcludedGroups="org.apache.drill.categories.SlowTest,org.apache.drill.categories.UnlikelyTest,org.apache.drill.categories.SecurityTest"; \
-    elif [ $PHASE = "build_checkstyle_protobuf" ]; then \
-      MAVEN_OPTS="-Xms1G -Xmx1G" mvn install --no-transfer-progress -Drat.skip=false -Dlicense.skip=false --batch-mode -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -DskipTests=true -Dmaven.javadoc.skip=true -Dmaven.source.skip=true && \
-      pushd protocol && mvn process-sources -P proto-compile && popd && \
-      mkdir contrib/native/client/build && pushd contrib/native/client/build && /usr/bin/cmake -G "Unix Makefiles" .. && make cpProtobufs && popd; \
+    if [ $PHASE = "tests" ]; then
+      echo Selecting parallel GC to minimise peak mem usage.
```

Comment:
Please avoid adding such comments in the ci output, they might confuse people that trying to figure out why their tests failing.